tpgtools/overrides/compute/beta/firewall_policy_association.yaml

```diff
@@ -0,0 +1,2 @@
+# Fine-grained resource, no sweeper
```

Comment:
Overrides will need to be duplicated to GA, we're getting sweepers at beta currently.